### ISSUE TITLE
RuntimeError: Failed to make system call

### ISSUE THREAD

**Issue by Johnsalzarulo** [titusfortner/webdrivers#162]
#### Summary
I've been stuck for hours trying to get my test suite to run on my local machine. I keep getting runtime errors. Details below. 

#### Debug Info
Please provide the following information for bug reports:

* Webdrivers version: `gem "webdrivers", "~> 4.0"`
* Ruby version: `ruby 2.5.0` 
* Operating system / CI Environment: `MacOS 10.14.4`
* Browser and version: `Chrome 78.0.3904.108`

#### Expected Behavior
I would expect my tests to run without `RuntimeError`  

#### Actual Behavior

````
RuntimeError:         RuntimeError: Failed to make system call: ["/Users/johnsalzarulo/.webdrivers/chromedriver", "--version"]
            test/support/authentication_helper.rb:4:in `log_in_user'
            test/controllers/library/content_imports/content_imports_controller_test.rb:20:in `block in <class:ContentImportsControllerTest>'

2019-12-04 12:09:09 DEBUG Webdrivers Checking current version
2019-12-04 12:09:09 DEBUG Webdrivers /Users/johnsalzarulo/.webdrivers/chromedriver is already downloaded
2019-12-04 12:09:09 DEBUG Webdrivers making System call: ["/Users/johnsalzarulo/.webdrivers/chromedriver", "--version"]
2019-12-04 12:09:09 DEBUG Webdrivers Checking current version
2019-12-04 12:09:09 DEBUG Webdrivers /Users/johnsalzarulo/.webdrivers/chromedriver is already downloaded
2019-12-04 12:09:09 DEBUG Webdrivers making System call: ["/Users/johnsalzarulo/.webdrivers/chromedriver", "--version"]
````

### What I've Tried 

1. I've found [this wiki link](https://github.com/titusfortner/webdrivers/wiki/Using-with-VCR-or-WebMock) and I've hit my head against the wall trying to update my `test_helper.rb` config accordingly. 

For example: 

````ruby
driver_urls = Webdrivers::Common.subclasses.map do |driver|
  Addressable::URI.parse(driver.base_url).host
end


WebMock.disable_net_connect!(allow_localhost: true, allow: driver_urls)

````

2. I've also gone round and round trying to even hardcode the driver_urls: 

````ruby
WebMock.disable_net_connect!(
  allow: [
    'localhost',
    '127.0.0.1',
    /selenium/,
    'chromedriver.storage.googleapis.com',
    'api.stripe.com',
    'https://chromedriver.storage.googleapis.com/LATEST_RELEASE',
    'chromedriver.storage.googleapis.com',
    'github.com',
    'selenium-release.storage.googleapis.com'
    ]
  )

````

3. I tried reboots 

4. I tried uninstalling and reisntalling Chrome 

5. I've tried reinstalling my chromedriver with `brew cask reinstall chromedriver` 

6. I've even tried disabling `Webmock` and I can't seem to get my tests to run. Just constant runtime erros. 

7. I tried updating my MacOS DNS servers, just in case there was something with networking that was funky. 

8. I've tried it on multiple internet connections, just to ensure I had no firewall or proxy shenanigans. 

**Does anyone have any thoughts of what else I can try? Thanks so much**

**Comment by Johnsalzarulo** [comment]
Couple updates: 

A. Confirmed I have `chromedriver` installed
<img width="582" alt="Screen Shot 2019-12-04 at 12 36 02 PM" src="https://user-images.githubusercontent.com/3476109/70179293-aa837680-1692-11ea-8295-f88e9e5fbdd5.png">

B. Manually hunted down where it's installed in my OS 
<img width="580" alt="Screen Shot 2019-12-04 at 12 34 32 PM" src="https://user-images.githubusercontent.com/3476109/70179323-b838fc00-1692-11ea-8ed9-02747b09a25c.png">

C. Added this to my paths 

<img width="567" alt="Screen Shot 2019-12-04 at 12 37 06 PM" src="https://user-images.githubusercontent.com/3476109/70179383-cedf5300-1692-11ea-85fb-970ffe08e0b7.png">

### Getting same results 😢

<img width="1046" alt="Screen Shot 2019-12-04 at 12 38 08 PM" src="https://user-images.githubusercontent.com/3476109/70179473-f6ceb680-1692-11ea-9d48-bdf76c4dc959.png">

**Comment by kapoorlakshya** [comment]
Hey @Johnsalzarulo, that error typically means that the gem was unable to execute the driver binary for some reason. It is unrelated to Webmock, DNS, or Chrome.

Can you undo step 5 (`brew install`) and step C (add to path), and then try to manually extract the version from the downloaded file? Maybe even delete the existing `chromedriver` file and redownload it using the gem before executing this:

```
Users/johnsalzarulo/.webdrivers/chromedriver --version
```

It should return version 78 dot something to match your Chrome version. If you're unable to execute it manually, whatever error you see in the terminal is what's causing the `RuntimeError`.

**Comment by Johnsalzarulo** [comment]
@kapoorlakshya 

Thank you so much for your support. Seriously people like you are what make the world a good place. 

I gave it a shot based on your suggestions with no luck. 

For some reason in trying to make things work, I tossed `gem "chromedriver-helper"` into my gemfile and this got things up and running. I see it's depreciated but at the moment — I'm productive again when I remove it things crash again. 

I'm closing the issue now, hopefully these breadcrumbs help others. 

As well — at some point, I should probably dig deeper into my system to solve the root problem. 

Thanks again so much. 👍
